src/main/resources/templates/kategoria.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"/>
+    <title>Zadania w wybranej kategorii</title>
+</head>
+<body>
+
+<div class="input-group-text">Wyszukaj zadań w wybranej kategorii:</div><br>
+<form class="input-group input-group-sm mb-3" method="post" action="/category">
+    <div class="input-group mb-3" style="margin-right: 50%">
+        <select class="custom-select" name="category">
```

Comment:
Tutaj zamiast wymieniania wszystkich elementów lepiej byłoby zastosować foreach i wypisać wszystkie dostępne elementy enuma. Wtedy w przypadku dodania kolejnej kategorii nie byłoby potrzeby zmian w tym miejscu.

Reply:
OK. Spróbuję to zmienić.